database/submit_to_database.py

```diff
@@ -284,7 +191,16 @@ def flux_cal_and_submit(time_detection, time_obs, metadata, bestprof_data,
                                     beg=beg, t_int=t_int)
 
     #estimate S/N
-    sn, u_sn, _, w_equiv_bins, _, w_equiv_ms, u_w_equiv_ms, scattered = snfe.analyse_pulse_prof(prof_data=profile, period=period, verbose=True)
+    prof_dict = prof_utils.analyse_pulse_prof(prof_data=profile, period=period, verbose=True)
+    sn = prof_dict["sn"]
+    u_sn = prof_dict["sn_e"]
+    w_equiv_bins = prof_dict["weq"]
+    u_w_equiv_bins =  prof_dict["weq_e"]
+    w_equiv_ms = period/num_bins * w_equiv_bins
+    u_w_equiv_ms = period/num_bins * u_w_equiv_bins
+    scattering = prof_dict["wscat"]
+    scattering_error = prof_dict["wscat_e"]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [local variable 'scattering_error' is assigned to but never used](https://app.codacy.com/manual/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4645369)